docs/network protocol.md

```diff
@@ -31,6 +31,9 @@ There are also a number of community-supported libraries available that implemen
 | GameMaker: Studio 2.x+        | [see Discord](https://discord.com/channels/731205301247803413/1166418532519653396)                 |                                                                                 |
 
 ## Synchronizing Items
+After a client connects, it will receive all previously collected items for its associated slot in a [ReceivedItems](#ReceivedItems) packet. This will include items the client may have already processed in a previous play session.  
```

Comment:
I would suggest `it will receive all previously received items`, instead of collected. It feels more intuitive to me, as collected seems to imply what the player actually **did**, so locations more than items.

Reply:
Now that I re-read this comment, I see it repeats the word "receive" twice.
Maybe `It will get all previously received items [...] in a ReceivedItems packet`?

Reply:
Hm, so, the reason I went for "collected" instead of "received" here is that "received" implies "received by the client", which is not true, it will just *send everything that was found for that slot*, which could include both "previously processed in the last play session" and "found while offline after the last play session" in one packet

Reply:
Maybe `found`? I'm not sure

Reply:
Since it's literally called a "ReceivedItems" packet, I personally still favor that word. In my mind, someone will have to understand what a received item is sooner or later anyway, so might as well rip the bandaid here.